app/models/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const process = require('process');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.json')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
+  .readdirSync(__dirname)
```

Comment:
So, you're reading all the models in the directory?

How many models are we expecting, such that this is easier than just listing the models directly?

Reply:
This is what sequelize generates when being setup, we can probably get rid of this and use our own initialization file (src/models/index.ts). I'm happy to add them manually in src/models/index.ts instead, where we would have the option to reference them and create the relations etc.

We'll just have to change the options for sequelize so it reads the models from there.

Reply:
If it's the generated version, let's just use it!